cssom-view-1/Overview.bs

```diff
@@ -1899,6 +1899,81 @@ When asked to <dfn export for=Document>run the resize steps</dfn> for a {{Docume
     <a attribute for=VisualViewport>width</a>, or <a attribute for=VisualViewport>height</a> properties changed since
     the last time these steps were run, <a>fire an event</a> named <a event>resize</a> at the {{VisualViewport}}.
 
+<h3 id=snap-events>Snap Events</h3>
+
+    This section integrates with the [[#scrolling-events|Scrolling events]].
+
+    Each {{Document}} associates each of its
+    <a spec=css-scroll-snap lt="scroll snap container">snap containers</a>
+    with at most one <dfn>snapchangingTargetBlock</dfn> and at most one
+    <dfn>snapchangingTargetInline</dfn> in the block and inline axes respectively.
+
+    Each {{Document}} associates each of its
+    <a spec=css-scroll-snap lt="scroll snap container">snap containers</a>
+    with at most one <dfn>snapchangedTargetBlock</dfn> and at most one
+    <dfn>snapchangedTargetInline</dfn> in the block and inline axes respectively.
```

Comment:
Duplicated paragraph?

Reply:
Ah, not duplicated (changed/changing). I think these could hang off the snap container itself tho, right? As in, the document doesn't _really_ do anything here.

Reply:
Yes it can be written without document, thanks for the suggestion. I've rewritten these without document.

Reply:
And thanks for taking a look!